domains/khag.json

```diff
@@ -0,0 +1,8 @@
+{
+    "owner": {
+        "username": "khag"
+    },
+    "records": {
+        "A": "103.77.240.59"
+    }
```

Comment:
A records should be in an array. File does not follow proper schema. See docs for more information: https://docs.is-a.dev/